jenkins/attach_agent.py

```diff
@@ -24,7 +24,7 @@
 SHARED = False
 
 INFO = {
-    'heavy': ('build unittest', 1, EXCLUSIVE),
+    'heavy': ('build unittest scalability', 1, EXCLUSIVE),
```

Comment:
Let's not do this until we also update agent-ctl.sh and/or attach_agent.py to be able to modify the set of tags on existing agents.